batavia/types/Dict.js

```diff
@@ -52,19 +52,30 @@ batavia.types.Dict = function() {
      **************************************************/
 
     Dict.prototype.__lt__ = function(other) {
-        return this.valueOf() < other;
+        throw new batavia.builtins.TypeError("unorderable types: dict() < dict()");
```

Comment:
What if `other` is an int or string or...?

This function should check for types and return the correct error message.

Alternatively, if you'd rather fill out all the types right now, throw the `NotImplemented` error if the type isn't `dict`.